src/EventStore.Core/DataStructures/ProbabilisticFilter/BloomFilterIntegrity.cs

```diff
@@ -4,12 +4,15 @@
 using EventStore.Core.Index.Hashes;
 using Serilog;
 
-namespace EventStore.Core.DataStructures.ProbabilisticFilter.MemoryMappedFileBloomFilter {
+namespace EventStore.Core.DataStructures.ProbabilisticFilter {
 	// This class is responsible for maintaining and validating a 4 byte hash at the end
 	// of the cache line.
 	public class BloomFilterIntegrity {
 		public const int CacheLineSize = 64;
 		public const int HashSize = 4;
+		//qq this is the flush unit, discuss with james whether it should be configurable and what the default should be.
+		public const int PageSize = 8 * 1024;
```

Comment:
it could also be looked up at runtime with `sysconf(_SC_PAGESIZE)` on linux and `GetSystemInfo()` on windows with a fallback of the usual default (4096)

Reply:
if it is configurable, the value 0 could be used for auto-configuration (similar to how it is for other auto-configurable options)

Reply:
note: if it is configurable, it should be a multiple of the cache line size (64) and disk block size (4096)